Dockerfile

```diff
@@ -2,13 +2,14 @@ FROM rust:buster
 
```

Comment:
is the docker container used? If so, has its version been updated?

Reply:
the docker container gets used for github actions, dockerhub has been update with this new version